packages/events/src/router/event/event.actions.correction.test.ts

```diff
@@ -84,6 +84,75 @@ test(`${ActionType.APPROVE_CORRECTION} validation error message contains all th
   ).rejects.matchSnapshot()
 })
 
+test(`${ActionType.REQUEST_CORRECTION} Allows passing fields that are conditionally removed`, async () => {
+  const { user, generator } = await setupTestCase()
+  const client = createTestClient(user)
+
+  const event = await client.event.create(generator.event.create())
+  event.id
+
+  const data = generator.event.actions.correction.request(event.id, {
+    data: {
+      'applicant.dob': '02-1-2024',
+      'applicant.firstname': 'John',
+      'applicant.surname': 'Doe',
+      'recommender.none': false
+    }
+  })
+
+  await expect(
+    client.event.actions.correction.request(data)
+  ).rejects.matchSnapshot()
+})
+
+test(`${ActionType.REQUEST_CORRECTION} Skips required field validation when they are conditionally hidden`, async () => {
```

Comment:
Cool, and I guess it would be an error if the client even tried to send a field that was logically impossible based on other fields?